Core/Src/state_machine.c

```diff
@@ -2,27 +2,25 @@
 #include "can_handler.h"
 #include "fault.h"
 #include "nero.h"
-#include "pdu.h"
 #include "queues.h"
 #include "monitor.h"
 #include "serial_monitor.h"
 #include "nero.h"
-#include "pdu.h"
 #include "queues.h"
+#include "pedals.h"
 #include <stdbool.h>
 #include <stdio.h>
+#include <stdlib.h>
+#include "cerb_utils.h"
 
 #define STATE_TRANS_QUEUE_SIZE 4
+#define STATE_TRANSITION_FLAG  1U
 
 /* Internal State of Vehicle */
 static state_t cerberus_state;
 
-static pdu_t *pdu;
-
 extern IWDG_HandleTypeDef hiwdg;
```

Comment:
do we still need to pet the watchdog here?

Reply:
I don't know why we ever were

Reply:
Remember we crashed the code in  order to recover from a fault.

Reply:
lol right